addons/core/.template-lintrc.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 module.exports = {
-  extends: ['recommended'],
+  extends: ['recommended', 'stylistic'],
   rules: {
     quotes: 'single',
-    'eol-last': 'editorconfig',
+    'self-closing-void-elements': false,
```

Comment:
This [rule](https://github.com/ember-template-lint/ember-template-lint/blob/v3.12.0/docs/rule/self-closing-void-elements.md) didn't seem that consequential and it conflicted with prettier so I turned it off.

Let me know if you think we should keep it on.